packages/main/src/components/AnalyticalTable/AnalyticalTable.stories.mdx

```diff
@@ -979,9 +982,9 @@ The prop expects a function with an optional parameter containing the `row` inst
       infiniteScrollThreshold: { table: { disable: true } },
       onSort: { table: { disable: true } },
       onGroup: { table: { disable: true } },
-      onRowSelected: { table: { disable: true } },
+      onRowSelect: { table: { disable: true } },
```

Comment:
Both the deprecated and new prop shouldn't be shown in the args table here and in the lines below.